app/models/tag_trend.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: tag_trends
+#
+#  id       :bigint(8)        not null, primary key
+#  tag_id   :bigint(8)        not null
+#  score    :float            default(0.0), not null
+#  rank     :integer          default(0), not null
+#  allowed  :boolean          default(FALSE), not null
+#  language :string
+#
+class TagTrend < ApplicationRecord
```

Comment:
This new model could use model spec ... separately, we might be able to pull out what I think is repeated coverage for the other classes that include `RankedTrend` into a shared example and use that in all of them.